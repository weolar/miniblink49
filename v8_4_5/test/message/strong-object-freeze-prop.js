// Copyright 2015 the V8 project authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// Flags: --strong-mode

"use strong";

let o = {};
Object.defineProperty(o, "foo", { writable: true });
Object.defineProperty(o, "foo", { writable: false });
