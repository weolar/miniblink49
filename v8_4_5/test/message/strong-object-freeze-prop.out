# Copyright 2015 the V8 project authors. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.
*%(basename)s:11: TypeError: On strong object #<Object>, redefining writable, non-configurable property 'foo' to be non-writable is deprecated
Object.defineProperty(o, "foo", { writable: false });
       ^
TypeError: On strong object #<Object>, redefining writable, non-configurable property 'foo' to be non-writable is deprecated
    at Function.defineProperty (native)
    at *%(basename)s:11:8
